backend/src/database/migrations/V1757490935__create_activityTypes_table.sql

```diff
@@ -0,0 +1,120 @@
+-- Insert Code Contributions activity types
+-- Git platform (Code Contributions)
+INSERT INTO "activityTypes" ("activityType", platform, "isCodeContribution", "isCollaboration") VALUES
+('authored-commit', 'git', true, false),
+('reviewed-commit', 'git', true, false),
+('tested-commit', 'git', true, false),
+('co-authored-commit', 'git', true, false),
+('informed-commit', 'git', true, false),
+('influenced-commit', 'git', true, false),
+('approved-commit', 'git', true, false),
+('committed-commit', 'git', true, false),
+('reported-commit', 'git', true, false),
+('resolved-commit', 'git', true, false),
+('signed-off-commit', 'git', true, false);
+
+-- GitHub platform (Code Contributions)
+INSERT INTO "activityTypes" ("activityType", platform, "isCodeContribution", "isCollaboration") VALUES
+('pull_request-opened', 'github', true, false),
+('pull_request-closed', 'github', true, false),
+('pull_request-review-requested', 'github', true, false),
+('pull_request-reviewed', 'github', true, false),
+('pull_request-merged', 'github', true, false),
+('pull_request-comment', 'github', true, false),
+('authored-commit', 'github', true, false);
+
+-- Gerrit platform (Code Contributions)
+INSERT INTO "activityTypes" ("activityType", platform, "isCodeContribution", "isCollaboration") VALUES
+('changeset-new', 'gerrit', true, false),
+('changeset-created', 'gerrit', true, false),
+('changeset-merged', 'gerrit', true, false),
+('changeset-closed', 'gerrit', true, false),
+('changeset-abandoned', 'gerrit', true, false),
+('changeset_comment-created', 'gerrit', true, false),
+('patchset-created', 'gerrit', true, false),
+('patchset_comment-created', 'gerrit', true, false),
+('patchset_approval-created', 'gerrit', true, false);
+
+-- GitLab platform (Code Contributions)
+INSERT INTO "activityTypes" ("activityType", platform, "isCodeContribution", "isCollaboration") VALUES
+('merge_request-opened', 'gitlab', true, false),
+('merge_request-closed', 'gitlab', true, false),
+('merge_request-review-requested', 'gitlab', true, false),
+('merge_request-review-approved', 'gitlab', true, false),
+('merge_request-review-changes-requested', 'gitlab', true, false),
+('merge_request-merged', 'gitlab', true, false),
+('merge_request-comment', 'gitlab', true, false);
+
+-- Insert Collaboration activity types
+-- GitHub platform (Collaboration)
+INSERT INTO "activityTypes" ("activityType", platform, "isCodeContribution", "isCollaboration") VALUES
+('discussion-started', 'github', false, true),
+('issues-opened', 'github', false, true),
+('issues-closed', 'github', false, true),
+('issue-comment', 'github', false, true),
+('discussion-comment', 'github', false, true);
+
+-- GitLab platform (Collaboration)
+INSERT INTO "activityTypes" ("activityType", platform, "isCodeContribution", "isCollaboration") VALUES
+('issues-opened', 'gitlab', false, true),
+('issues-closed', 'gitlab', false, true),
+('issue-comment', 'gitlab', false, true),
+('authored-commit', 'gitlab', false, true);
```

Comment:
`authored-commit ` is categorised as a code contribution for Git and GitHub but as a collaboration for GitLab. It doesn't seem right. Let's double check this before merging.